src/functional/ffront/fbuiltins.py

```diff
@@ -61,10 +61,10 @@ def __gt_type__(self):
 BUILTINS = {name: globals()[name] for name in __all__ + ["bool", "int", "float"]}
 
 
-# TODO(ricoh): This should be reunified with ``iterator.runtime.Offset``
-# potentially in ``functional.common``, which requires lifting of
-# ``ffront.common_types`` into ``functional``.
-# TODO(tehrengruber): remove inheritance since this is conceptually different to itir offset
+# TODO(tehrengruber): FieldOffset and runtime.Offset are not an exact conceptual
+#  match. Revisit if we want to continue subclassing here. If we split
+#  them also check whether Dimension should continue to subclass or define
```

Comment:
Tiny optional improvement before merging: `Dimension` is not subclassing anymore.